packages/cli/src/cli/github/make.cog.pr.ts

```diff
@@ -26,23 +35,39 @@ export class MakeCogGithub extends Github {
   /**
    * Prepare and create pull request for the aerial tileset config
    */
-  async updateRasterTileSet(filename: string, layer: ConfigLayer, category: Category): Promise<void> {
+  async updateRasterTileSet(
+    filename: string,
+    layer: ConfigLayer,
+    category: Category,
+    addToAerial: boolean,
+  ): Promise<void> {
     const branch = `feat/bot-config-raster-${this.imagery}`;
 
     // Clone the basemaps-config repo and checkout branch
     this.clone();
     this.configUser();
     this.getBranch(branch);
 
-    // Prepare new aerial tileset config
     this.logger.info({ imagery: this.imagery }, 'GitHub: Get the master TileSet config file');
-    const tileSetPath = fsa.joinAll(this.repoName, 'config', 'tileset', `${filename}.json`);
-    const tileSet = await fsa.readJson<ConfigTileSetRaster>(tileSetPath);
-    const newTileSet = await this.prepareRasterTileSetConfig(layer, tileSet, category);
-
-    // skip pull request if not an urban or rural imagery
-    if (newTileSet == null) return;
-    await fsa.write(tileSetPath, await this.formatConfigFile(tileSetPath, newTileSet));
+    if (addToAerial) {
```

Comment:
This is not always going to be `aerial` maybe we should call it `--individual` ?